src/riscv64_qemuvirt_main.rs

```diff
@@ -34,8 +33,15 @@ pub type InterruptsImpl = arch::riscv64::interrupts::Interrupts;
 pub type MemoryImpl = arch::riscv64::sv39::PageTable;
 pub type ConsoleImpl = Ns16550;
 
+pub const UART_ADDR: usize = 0x1000_0000;
```

Comment:
I liked the original qemu virt names

Reply:
Since we are inside the QEMU main, what's the point of prefixing them?